docs/sign-up.rst

```diff
@@ -0,0 +1,72 @@
+.. _sign-up:
+
+
+===================================
+Create a CrateDB Cloud user account
+===================================
+
+This page shows you how to:
+
+* Create a user account
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _create-account:
+
+Create an account
+=================
+
+Call up the `CrateDB Cloud sign-up page`_ and complete the form.
+
+.. image:: _assets/img/cloud-sign-up-page.png
+
+Once you've completed the form, select *Sign up*.
+
+You'll receive a confirmation email providing your verification code.
+
+Open your email inbox and open the verification email.
+
+.. NOTE::
+
+  If you didn't receive the verification email, select *Resend it* to request a
+  new verification email.
+
+Copy your verification code from the email.
+
+.. image:: _assets/img/confirmation-email-received.png
+
+Paste your verification code into the corresponding field on the sign-up page
```

Comment:
try to condense the email verification instructions. but do include the image that comes after this